File: executable/node/Greys-heatmap.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x'

var unpack = require('ndarray-unpack');
var zeros = require('zeros');
var fill = require('ndarray-fill');

var size = 50;
var z = unpack(fill(zeros([size, size]), function(c,r) {
  return Math.sqrt((r*c)/(size*size));
}));

var data = [
  {
    z: z, 
    scl: "Greys", 
    type: "heatmap"
  }
];
var layout = {title: "Greys"};

File: executable/r/Greys-heatmap.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

size <- 50
z <- matrix(0, size, size)
for(r in 1:size) {
    for(c in 1:size) {
        z[r, c] <- sqrt(r*c/size^2)
    }
}

trace1 <- list(
  z = z, 
  scl = "Greys", 
  type = "heatmap"
)

layout <- list(title = "Greys")



response <- p$plotly(trace0, kwargs=list(layout=layout, filename="Greys-heatmap", fileopt="overwrite"))
url <- response$url
filename <- response$filename